TLSharp.Core/TelegramClient.cs

```diff
@@ -51,9 +64,10 @@ public Session Session
             this.apiHash = apiHash;
             this.apiId = apiId;
             this.handler = handler;
+            this.dcIpVersion = dcIpVersion;
 
             session = Session.TryLoadOrCreateNew(store, sessionUserId);
-            transport = new TcpTransport (session.DataCenter.Address, session.DataCenter.Port, this.handler);
+            transport = new TcpTransport(session.DataCenter.Address, session.DataCenter.Port, this.handler);
```

Comment:
no need to change this line above ^

Reply:
when i swapped the editor to replace tabs with the spaces, i had to autoformat and this was auto formatted as well.

Reply:
well then revert the autoformat, you're breaking the git-blame for the lines that don't need changing